torch/csrc/jit/codegen/cuda/predicate_compute.cpp

```diff
@@ -25,18 +29,21 @@ std::vector<kir::Bool*> PredicateCompute::computePredicates(
     return {};
   }
 
+  std::vector<kir::Bool*> preds(root.size(), new kir::Bool(true));
```

Comment:
clang-tidy?

Reply:
I just wanted to pre-initialize before the loop below.